app/assets/stylesheets/dashboard/epic_queues.sass

```diff
@@ -27,3 +27,14 @@
   cursor: pointer
 th.text-primary
   color: #fff
+
+.epic-queue-records-table
+
+  .black-note
+    color: black
+
+  .blue-note
+    color: blue
```

Comment:
This should probably be the $sparc-blue theme color. You can access it by importing `proper/colors` at the top of the sass file